CMakeLists.txt

```diff
@@ -52,6 +81,16 @@ endif()
 add_subdirectory(lib)
 
 # External libraries
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
Why is this branch necessary?  It seems like the below `find_package` line works fine with the macOS CI runners and `brew`.  Are you trying to avoid the need for the `brew reinstall` step in the CI run?